.prettierignore

```diff
@@ -0,0 +1,6 @@
+build
```

Comment:
Could you change this file to match the structure of this repo? Running the prettier command after a build throws up errors about the dist/* directory containing code style issues.